python/nav/models/sql/changes/sc.05.06.0001.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO eventtype (eventtypeid,eventtypedesc,stateful) VALUES
+  ('juniperYellowAlarmState','Tells us if a Juniper device has any open yellow alarms.','y');
+INSERT INTO eventtype (eventtypeid,eventtypedesc,stateful) VALUES
+  ('juniperRedAlarmState','Tells us if a Juniper device has any open red alarms.','y');
+INSERT INTO alerttype (eventtypeid,alerttype,alerttypedesc) VALUES
+  ('juniperYellowAlarmState','juniperYellowAlarmOn','The Juniper device has some yellow alarms.');
+INSERT INTO alerttype (eventtypeid,alerttype,alerttypedesc) VALUES
+  ('juniperYellowAlarmState','juniperYellowAlarmOff','The Juniper device has no yellow alarms.');
+INSERT INTO alerttype (eventtypeid,alerttype,alerttypedesc) VALUES
+  ('juniperRedAlarmState','juniperRedAlarmOn','The Juniper device has some red alarms.');
+INSERT INTO alerttype (eventtypeid,alerttype,alerttypedesc) VALUES
+  ('juniperRedAlarmState','juniperRedAlarmOff','The Juniper device has no red alarms.');
```

Comment:
The 5.6 release came and went without this PR, so in keeping with the naming scheme, this should be renamed to `sc.05.07.0001.sql` (yeah, I know, the naming scheme is stupid) for release in 5.7.